toolchain.mk

```diff
@@ -40,7 +40,7 @@ aarch32:
 aarch64:
 	$(call dltc,$(AARCH64_PATH),$(SRC_AARCH64_GCC),$(AARCH64_GCC_VERSION))
 
-CLANG_PATH			?= $(ROOT)/clang-9.0.1
+CLANG_PATH			?= $(ROOT)/clang-12.0.0
```

Comment:
maybe pass version to get_clang.sh in `dl-clang` below?

Reply:
Like so? https://github.com/OP-TEE/build/pull/471